client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -396,7 +397,7 @@ export function YourBrowserSupportToolkit() {
       </header>
       <p className="article-lead">
         In this article discover the resources available, to help you develop a
-        site that will perform well across browsers and devices
+        site that will perform well across browsers and devices.
```

Comment:
We're not using periods at the end of the article lead, so no period was correct.